pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -87,6 +93,11 @@ buildPythonPackage rec {
     "test_collinear_kgks_gga"
     "test_libxc_gga_deriv4"
     "test_sacasscf_grad"
+    "test_rdm_trace"
```

Comment:
Those are also fixed by #420916 and I think the latter two don't need to be disabled with its fix.

Reply:
Looks like #420916 will close down three of my PRs. The `plotly` and `chart-studio` PR is still viable.